src/MIDebugPackage/MIDebugPackage.csproj

```diff
@@ -215,6 +215,59 @@
       <TranslationFile>$(MIEngineRoot)loc\LCL\{Lang}\lib\OpenFolderSchema.json.lcl</TranslationFile>
     </FilesToLocalize>
   </ItemGroup>
+  <Target Name="DropLocalizeDir" AfterTargets="Localize;DropFiles" Condition="'$(LocalizationEnabled)'=='true'">
+    <PropertyGroup>
+      <LocalizedDropDir>$(DropRootDir)\loc\</LocalizedDropDir>
+      <LocalizeBinDir>$(OutDir)localize\</LocalizeBinDir>
+    </PropertyGroup>
+    <ItemGroup>
+      <Locale Include="$(LocalizeBinDir)ENU\*.json">
+        <LCID>1033</LCID>
+      </Locale>
+      <Locale Include="$(LocalizeBinDir)CSY\*.json">
+        <LCID>1029</LCID>
+      </Locale>
+      <Locale Include="$(LocalizeBinDir)DEU\*.json">
+        <LCID>1031</LCID>
+      </Locale>
+      <Locale Include="$(LocalizeBinDir)ESN\*.json">
+        <LCID>3082</LCID>
+      </Locale>
+      <Locale Include="$(LocalizeBinDir)FRA\*.json">
+        <LCID>1036</LCID>
+      </Locale>
+      <Locale Include="$(LocalizeBinDir)ITA\*.json">
+        <LCID>1040</LCID>
+      </Locale>
+      <Locale Include="$(LocalizeBinDir)JPN\*.json">
+        <LCID>1041</LCID>
+      </Locale>
+      <Locale Include="$(LocalizeBinDir)KOR\*.json">
+        <LCID>1042</LCID>
+      </Locale>
+      <Locale Include="$(LocalizeBinDir)PLK\*.json">
+        <LCID>1045</LCID>
+      </Locale>
+      <Locale Include="$(LocalizeBinDir)PTB\*.json">
+        <LCID>1046</LCID>
+      </Locale>
+      <Locale Include="$(LocalizeBinDir)RUS\*.json">
+        <LCID>1049</LCID>
+      </Locale>
+      <Locale Include="$(LocalizeBinDir)TRK\*.json">
+        <LCID>1055</LCID>
+      </Locale>
+      <Locale Include="$(LocalizeBinDir)CHS\*.json">
+        <LCID>2052</LCID>
+      </Locale>
+      <Locale Include="$(LocalizeBinDir)CHT\*.json">
+        <LCID>1028</LCID>
+      </Locale>
+    </ItemGroup>
+    <MakeDir Condition="!Exists($(LocalizedDropDir))" Directories="$(LocalizedDropDir)" />
```

Comment:
Do we need to create the various subdirectories also? Or will Copy do that for us? If Copy will do it, can we remove this step entirely?